apps/fair/client/index.coffee

```diff
@@ -87,3 +86,6 @@ module.exports.init = ->
           "&profile_id=#{profile.id}" +
           "&fair_id=#{fair.id}" +
           "&fair_name=#{fair.get('name')}"
+
+  # Add marketing signup CTA modal
+  new MarketingSignupModal
```

Comment:
Wow, what does it mean in CoffeeScript for a line like this to be at the end of a file on its own?